packages/toaster/tests/Toaster.test.tsx

```diff
@@ -33,17 +33,19 @@ describe("Toaster", () => {
     expect(component.state("toasts")).not.toContain(testToast[0].props.id);
   });
 
-  it("dismisses toasts after specified time", () => {
```

Comment:
this test was broken and doesn't seem valid. We only want toasts automatically removed if `autodismiss=true` which we have a different test for.